pyprime/.gitattributes

```diff
@@ -0,0 +1,6 @@
+* text=auto
+
+*.fmd binary
+*.pmdat binary
+*.scdoc binary
+*.stp binary
```

Comment:
@ninad-kamat let's ensure that all files being added in the future have lower-case extensions, i.e. in case of files such as TESTCASE.STP we should rename those first.

Reply:
Sure @ansutremel . In fact everything in this repo is snake_case.